Formula/openj9.rb

```diff
@@ -0,0 +1,113 @@
+class Openj9 < Formula
+  desc "High performance, scalable, Java virtual machine"
+  homepage "https://www.eclipse.org/openj9/"
+  url "https://github.com/eclipse/openj9/archive/refs/tags/openj9-0.25.0.tar.gz",
+    using: :nounzip
```

Comment:
```suggestion
  url "https://github.com/eclipse/openj9/archive/refs/tags/openj9-0.25.0.tar.gz", using: :nounzip
```
Let's follow the style of other formulae here.